apps/strava/strava.star

```diff
@@ -414,14 +405,20 @@ def athlete_stats(config, refresh_token, period, sport, units):
             "Authorization": "Bearer %s" % access_token,
         }
 
-        cache_prefix = "%s/%s/%s/" % (refresh_token, sport, period)
-
         # Get logged in athlete
         athlete = cache.get("%s/athlete_id" % refresh_token)
         if not athlete:
-            print("Getting athlete ID from API, access_token was cached.")
             url = "%s/athlete" % STRAVA_BASE
-            response = http_get(url, headers = headers)
+
+            # Note - we're double caching the athlete ID here since it should
```

Comment:
The athlete ID didn't map cleanly to our new client - given the app sets it when getting an access token. So I opted to double cache it, which feels fine since I don't see how this would change